include/FreeRTOS.h

```diff
@@ -3032,6 +3032,18 @@
     #define configCONTROL_INFINITE_LOOP()
 #endif
 
+/* Set configENABLE_PAC and/or configENABLE_BTI to 1 to enable PAC and/or BTI
+ * support and 0 to disable them. These are currently used in ARMv8.1-M ports. */
+#if ( portHAS_PACBTI_FEATURE == 1 )
```

Comment:
This code is not required because we already have an #error in case configENABLE_PAC and configENABLE_PAC are not defined. This means that an M85 application must define these constants in their FreeRTOSConfig.h.

Reply:
Hi @aggarg,
I've applied your suggestion.
Thanks!